lib/src/collect.dart

```diff
@@ -11,7 +11,7 @@ import 'util.dart';
 import 'hitmap.dart';
 
 const _retryInterval = Duration(milliseconds: 200);
-const _debugTokenPositions = false;
+const _debugTokenPositions = bool.fromEnvironment('DEBUG_COVERAGE');
```

Comment:
Then you'll want to document this – in the README at a minimum.

And maybe a test 😁

Reply:
I don't think users should ever use this flag. It's for us to debug coverage problems.

Reply:
Ack...